components/about_build_modal/about_build_modal_cloud/about_build_modal_cloud.scss

```diff
@@ -0,0 +1,72 @@
+.about-modal.cloud {
+    position: absolute;
+    margin: auto;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%) !important;
+    .modal-content{
+        overflow: hidden;
+        border: 1px solid rgba(var(--center-channel-color-rgb), 0.16);
+        box-shadow: 0px 20px 32px rgba(0, 0, 0, 0.12);
+        border-radius: 8px;
+
+        .modal-header {
+            width: 600px;
+            height: 66px;
+        }
+        .modal-body {
+            padding-top: 0px;
+            height: 294px;
+            width: 600px;
+            .about-modal__logo {
+                padding-left: 0;
+                padding-right: 32px;
+                svg {
+                    height: 104px;
+                    width: 104px;
+                }
+            }
+            .about-modal__title {
+                margin: 0;
+                font-weight: 600;
+                font-size: 24px;
+                line-height: 32px;
+            }
+            .about-modal__content {
+                padding-top: 46px;
+
+            }
+            .subtitle {
+                width: 294px;
+                height: 24px;
+                left: 0px;
+                top: 32px;
+                font-size: 16px;
+                line-height: 24px;
+                color: #3D3C40;
```

Comment:
We should use the theme color here and may be what Matt was referring to in dark mode.

Reply:
Yeah definitely the culprit